src/components/Simulation/SimulationBlocks.js

```diff
@@ -40,10 +39,23 @@ const SimulationBlocks = props => {
     return <p>{error}</p>;
   }
 
-  return Object.keys(setsByCode).map(key => {
-    const periodViews = setsByCode[key];
-    return <SimulationBlock key={key} title={key} periodViews={periodViews} />;
-  });
+  // Placeholder before future split async fetch of Periodviews
+  // At least now the list can be filtered by code from url params
+  // Which mean we can link to that url from the set edit page
+  // Ex:
+  // ?sets=cospro__b_1__gestion_manuel,cospro__a_4__reduction_des_frais
+
+  const displayedSetCodes = (props.searchQuery.sets || "")
+    .split(",")
+    .filter(i => i);
+
+  const sets = Object.keys(setsByCode);
+  const filteredSets = displayedSetCodes.length
```

Comment:
Don't think we need the ternary, since displayedSetCodes will be at least an empty array, so we can also filter on that.

Reply:
 But we want that if no url param is there to display the full list

Reply:
Got it, nevermind :)